.gitignore

```diff
@@ -319,3 +319,6 @@ src/build
 
 # Code coverage
 .coverage
+
+# remove later
+PnPCert
```

Comment:
When is later?

Reply:
"When will then be now?" "Soon." I have a PR on my fork for this.